src/Xamarin.Android.Build.Tasks/Tasks/GenerateLibraryResources.cs

```diff
@@ -100,85 +100,6 @@ class Package
 		/// <summary>
 		/// NOTE: all file paths used in this method should be full paths. (Or use AsyncTask.WorkingDirectory)
 		/// </summary>
-		void GenerateJava (Package package)
```

Comment:
@jonpryor I think you missed the summary above ^^

Reply:
I don't understand this comment.

Reply:
This comment is regarding the C# comment `<summary/>`.  It has been moved to be with the new location for the `GenerateJava()` method.